core/src/main/java/com/linecorp/armeria/common/logging/RequestLog.java

```diff
@@ -52,6 +53,11 @@
  */
 public interface RequestLog {
 
+    /**
+     * Returns the list of child {@link RequestLog}s. The lower the index is, the earlier it is added.
```

Comment:
Can remove `is`, though how about `list of {@link RequestLog}s, ordered by the time it was added.`?